code/modules/networks/computer3/computer3.dm

```diff
@@ -2,7 +2,7 @@
 
 /obj/machinery/computer3
 	name = "computer"
-	desc = "A computer workstation."
+	desc = "A computer that uses the bleeding-edge command line OS ThinkDos."
```

Comment:
```suggestion
	desc = "A computer that uses the bleeding-edge command line OS ThinkDOS."
```

Reply:
I'm on my phone and can't really edit things too easily, but wouldn't it be better to say "a computer that uses bleeding edge command line OS" instead of "the bleeding edge command line one"

Reply:
```suggestion
	desc = "A computer that uses the bleeding-edge command line ThinkDOS."
```

Reply:
that makes no grammatical sense